rcp/org.testeditor.rcp4.views.tcltestrun.tests/src/org/testeditor/rcp4/views/tcltestrun/model/TestExecutionManagerTest.xtend

```diff
@@ -10,6 +10,18 @@
  * akquinet AG
  * itemis AG
  *******************************************************************************/
+/*******************************************************************************
```

Comment:
? (but, it reminds me of checking my pending commits on missing (c) statements ...)